ngraph_bridge/CMakeLists.txt

```diff
@@ -176,6 +188,26 @@ else()
 	install(DIRECTORY ${NGRAPH_INSTALL_DIR}/lib/ DESTINATION "${NGTF_INSTALL_LIB_DIR}" )
 endif()
 
+# Install OpenVINO libraries
+if(ENABLE_OPENVINO)
+    set(IE_LIBS ${OPENVINO_ARTIFACTS_DIR}/deployment_tools/inference_engine/lib/intel64/)
+    set(TBB_LIBS ${OPENVINO_ARTIFACTS_DIR}/deployment_tools/inference_engine/external/tbb/lib/)
+    install(DIRECTORY ${IE_LIBS} DESTINATION "${NGTF_INSTALL_LIB_DIR}" )
+    install(DIRECTORY ${TBB_LIBS} DESTINATION "${NGTF_INSTALL_LIB_DIR}" )
+    if(USE_PREBUILT_OPENVINO)
+        file(GLOB NGRAPH_LIB_FILES "${IE_LIBS}/lib*" "${TBB_LIBS}/lib*")
+        foreach(LIB_FILE ${NGRAPH_LIB_FILES})
+            get_filename_component(lib_file_real_path ${LIB_FILE} ABSOLUTE)
+            get_filename_component(lib_file_name ${LIB_FILE} NAME)
+            if(${lib_file_name} MATCHES ".so*")
+                install(CODE "
+                execute_process(COMMAND patchelf --set-rpath $ORIGIN ${lib_file_real_path} COMMAND_ECHO STDOUT)
```

Comment:
Need to update README with the instructions and the dependencies.

Reply:
Done.